docs/content/index.mdx

```diff
@@ -16,12 +16,12 @@ Powertools dependencies are available in Maven Central. You can use your favouri
     <dependency>
         <groupId>software.amazon.lambda</groupId>
         <artifactId>powertools-tracing</artifactId>
-        <version>0.3.1-beta</version>
+        <version>0.4.0-beta</version>
     </dependency>
     <dependency>
         <groupId>software.amazon.lambda</groupId>
         <artifactId>powertools-logging</artifactId>
-        <version>0.3.1-beta</version>
+        <version>0.4.0-beta</version>
     </dependency>
     ...
```

Comment:
metrics is not there? normal?

Reply:
Good catch. I guess we missed updating it when we launched metrics module 😉